apertium-ibo.ibo.lexc

```diff
@@ -42,6 +50,10 @@ LEXICON N1
 
 %<n%>:  # ; 
 
+LEXICON PRN
+
+%<prn%>: # ;
+
 LEXICON V
 
 %<v%>%<inf%>%[%+inf%]: # ;
```

Comment:
If Igbo has more than a couple prefixes, then it might make sense to move to `lexd`.  Now would be a good time to do it, since there's so little morphology here so far.

Reply:
I could do the conversion in ~20 minutes from its present state if that would be desirable.